documentation/asciidoc/topics/proc_configuring_grafana_dashboards.adoc

```diff
@@ -0,0 +1,37 @@
+[id='configuring-grafana-dashboards_{context}']
+= Configuring {brandname} Dashboards
+{ispn_operator} provides global configuration properties that let you configure Grafana dashboards for {brandname} clusters.
+
+.Procedure
+
+. Create a `ConfigMap` named `infinispan-operator-config`.
++
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml/infinispan_operator_config.yaml[]
+----
++
+. Specify a namespace that {ispn_operator} watches as the value for the `grafana.dashboard.namespace` property.
```

Comment:
Operator must watch over the namespace where the Grafana Operator is running. I'd list it as prerequisite